jboss/container/wildfly/s2i/bash/artifacts/opt/jboss/container/wildfly/s2i/galleon/s2i_galleon

```diff
@@ -184,15 +240,20 @@ function galleon_provision_server() {
     fi
 
     if [ $nb -gt 1 ]; then
-      echo "Error, GALLEON_PROVISION_SERVER or GALLEON_PROVISION_LAYERS can't be used at the same time."
+      log_error "Error, GALLEON_PROVISION_SERVER or GALLEON_PROVISION_LAYERS can't be used at the same time."
```

Comment:
Probably better written as '"Error, GALLEON_PROVISION_SERVER **and** GALLEON_PROVISION_LAYERS can't be used at the same time."